File: VB/Applications/WpfLog4NetVBNoDI/DataStores/MainControlsDataStore.vb
Imports LogViewerVB.Core
Imports LogDataStore = LogViewerVB.Wpf.Logging.LogDataStore

Namespace DataStores

    ' Application-wide shared instance of the LogDataStore logging entries
    Public Module MainControlsDataStore

        Public ReadOnly Property DataStore As ILogDataStore = New LogDataStore()

    End Module

End Namespace

File: CSharp/Applications/WpfLog4NetNoDI/DataStores/MainControlsDataStore.cs
using LogViewer.Core;
using LogDataStore = LogViewer.Wpf.Logging.LogDataStore;

namespace WpfLog4NetNoDI.DataStores;

// Application-wide shared instance of the LogDataStore logging entries
public static class MainControlsDataStore
{
    public static ILogDataStore DataStore { get; } = new LogDataStore();
}